### PR TITLE
apachetop: update to 0.23.2

### PR DESCRIPTION
**Pull request by tessus**
#### Description

apachetop: update to 0.23.2

###### Type(s)

- [x] update

###### Tested on
macOS 10.14.6 18G9323
Xcode 11.3.1 11C504

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by tessus** [comment]
I don't have macOS 11 or 12, so I don't know what the issue is exactly. It runs perfectly on my 10.14 and 10.15.

Can someone with access to macOS 11 or 12 give me a hand?

File: sysutils/apachetop/Portfile [hunk 1]
```diff
@@ -16,10 +16,11 @@ long_description \
 platforms       darwin freebsd
 license         BSD
 
-depends_lib     port:readline port:pcre port:adns
+depends_lib     port:readline port:pcre2 port:adns
```
[the remark this answers is not in the document]
Comment (tessus) [reply sysutils/apachetop/Portfile]: Thanks, unfortunately this didn't help. What's strange is that I never had to add it.

Without having access to the config.log, I won't be able to figure out what the exact issue is. I might be able to install macOS 11 or 12 in a VM.

**Comment by tessus** [comment]
I have setup a VM with macOS 12 and `port build apachetop` works without errors.

I am not sure what is going on, but if I can build it on a freshly setup macOS, it stands to reason that the build machines are busted (or are setup differently).

```
macOS 12.6.3 21G419
Xcode 14.0.1 14A400
```

The config log is here: https://evermeet.cx/pub/logs/config.log

File: sysutils/apachetop/Portfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (tessus) [reply sysutils/apachetop/Portfile]: please see comment here https://github.com/macports/macports-ports/pull/17719#issuecomment-1433897543

**Comment by tessus** [comment]
Can someone with access to the build machines tell me what is going on?

As I mentioned before, it builds just fine on a newly installed macOS 12. It makes no sense that the buildbot reports an error.

**Comment by reneeotten** [comment]
You can check the CI logs to try and find out what the problem is. It looks like it's trying to use "pkgconfig" but it doesn't look like you have added that as a build dependency - perhaps that helps?

**Comment by tessus** [comment]
@reneeotten thanks, I'll try that too. I still don't understand why it would build in my VM. I did not do a `port install pkgconfig`. Very mysterious.

**Comment by tessus** [comment]
Closing and creating a new one to trigger the action again. Due to the gh issues there's no actions log.

**Pull request closed without merging**